src/upgrades/mod.rs

```diff
@@ -94,10 +106,8 @@ pub trait UpgradeStore {
 }
 
 /// Should be called when Krill starts, before the KrillServer is initiated
-pub fn pre_start_upgrade(work_dir: &PathBuf) -> Result<(), UpgradeError> {
-    upgrade_pre_0_6_0_cas_commands(work_dir)?;
-    upgrade_pre_0_6_0_pubd_commands(work_dir)?;
-    upgrade_pre_fix_info_0_8_0(work_dir)
```

Comment:
Given that this version of Krill will upgrade any version >= 0.6.0, why is this 0_8_0 fn no longer invoked? Is the logic from that fn now incorporated into the new 0_9_0 fn?